README.md

```diff
@@ -885,6 +887,47 @@ If these failures occur, add the warning to the list of permitted warnings found
 in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
 
 
+## Installing Python 3.8.3 on macOS BigSur
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+macOS Big Sur users may encounter a `build failed` error when installing Python through pyenv. 
+To remedy this there is a patch file which runs the installer compatible with 
+macOS Big Sur (11.x).
+
+First is to ensure that all requirements for the Python build environment from the pyenv [wiki](https://github.com/pyenv/pyenv/wiki#suggested-build-environment) 
+are met. We extend their suggestion and also install bzip2.
+```
+brew install openssl readline sqlite3 xz zlib bzip2
+```
+
+Follow any additional steps that `brew` requires for installation, these 
+should include modifying path variables `LDFLAGS` and `CPPFLAGS`
+The commands from the brew output to modify the aforementioned path variables 
+can be placed in the `~/.bash_profile` to make the change persistent
+
+Install python 3.8.3 using pyenv by running:
+```
+pyenv install --patch 3.8.3 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch)
+```
+
+
+## Failure to locate wheels on macOS Big Sur
+
+[Resolution source](https://stackoverflow.com/a/63972598)
+[macOS Big Sur Release Notes](https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-release-notes#Third-Party-Apps)
+
+Pip will not be able to locate the appropriate wheels due to the major release version of 
+Big Sur being incremented from `10.x` to `11.x`, instead pip will attempt to compile wheels 
+manually for wheels that it cannot locate. 
+
+In order to be able to run `make requirements` successfully, a backwards compatibility
+flag needs to be added to the `environment.local` file in the project root:
```

Comment:
I was wrong in the last comment, it should be `environment.local.py` file in the project root; `environment.local` does not exist, and setting the flag there will not work with our environment commands.